package.json

```diff
@@ -18,9 +18,9 @@
     "posttest:integration": "pnpm clean:integration",
     "playwright:integration": "playwright test --config ./integration/playwright.config.ts",
     "changeset": "changeset",
-    "version": "changeset version && node ./scripts/remove-prerelease-changelogs.mjs",
+    "changeset:version": "changeset version && node ./scripts/remove-prerelease-changelogs.mjs",
     "publish": "node scripts/publish.js",
-    "version:experimental": "node ./scripts/version experimental",
+    "version": "node ./scripts/version",
```

Comment:
Is it true we _only_ use this for experimental releases now via `pnpm run version 0.0.0-experimental-[sha] --skip-prompt`?

Reply:
Currently, but this is matching the structure in Remix where it's also used for nightly releases.

Reply:
Oh, and it's also referenced in `contributing.md`.